### DISCUSSION TITLE
Background map image not show up

### DISCUSSION THREAD

**Discussion by bingjiez** [iTwin/itwinjs-core#4844] [Q&A] [answered]
Hi, 

I set the view flag of backgroundMap to be true but the background still uses the color definition. Does anybody know what might cause this issue? I found an example from [this sandbox](https://www.itwinjs.org/sandboxes/iTwinPlatform/View%20Attributes) on iModel **House**

Thanks!

**Comment by MarcBedard8** [comment]
Be sure to provide a valid bing map key in your map layer options.
export const mapLayerOptions: MapLayerOptions = {
  // Api key for Bing Maps service. For more information https://www.microsoft.com/en-us/maps/create-a-bing-maps-key
  BingMaps: {
    key: "key",
    value: process.env.IMJS_BING_MAPS_KEY || "",
  },

**Comment by pmconne** [comment, marked as answer]
Also ensure your iModel is [geolocated](https://www.itwinjs.org/learning/geolocation/). The map cannot be displayed if we don't know where your iModel is located on the surface of the Earth.
